### PR TITLE
Added support for Urdu language

### PR DESCRIPTION
**Pull request by abduljalilm94**

### CODE REVIEW

File: test/definitions/ur.csv [hunk 1]
```diff
@@ -0,0 +1,18 @@
+0$0 ملی سیکنڈ
+1$1 ملی سیکنڈ
```
Comment (EvanHahn) [inline test/definitions/ur.csv]: [Google Translate](https://translate.google.com/#ur/en/1%20%D9%85%D9%84%DB%8C%20%D8%B3%DB%8C%DA%A9%D9%86%DA%88) says this means "1 millimeter", not "1 millisecond". Is this correct?

File: HISTORY.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+3.14.0 / 2018-03-23
```
Comment (EvanHahn) [inline HISTORY.md]: Could you change this header to "UNRELEASED"?

Comment (abduljalilm94) [reply HISTORY.md]: Sure! I will change this right away.

**Comment by EvanHahn** [comment]
Released in `humanize-duration@3.14.0`. Thanks!

**Pull request merged**